src/components/forms/TextArea/TextArea.tsx

```diff
@@ -135,6 +143,8 @@ TextArea.propTypes = {
   isDisabled: PropTypes.bool,
   value: PropTypes.string,
   defaultValue: PropTypes.string,
+  style: PropTypes.shape({}),
```

Comment:
what does styles used for here?

Reply:
it's just inline CSS style prop if we need to use it